resource/resource_graph.go

```diff
@@ -234,8 +234,8 @@ func (g *Graph) FindNodesByAPI(api API) []Name {
 	return ret
 }
 
-// findNodesByShortName returns all resources matching the given short name.
-func (g *Graph) findNodesByShortName(name string) []Name {
+// FindNodesByShortName returns all resources matching the given short name.
+func (g *Graph) FindNodesByShortName(name string) []Name {
```

Comment:
Exported `findNodesByShortName` to access it from `local_robot.go#getOptionalDependencies`.